server/app/views/admin/migration/AdminImportView.java

```diff
@@ -24,22 +24,22 @@
 import views.admin.AdminLayoutFactory;
 import views.components.FieldWithLabel;
 import views.components.Icons;
+import views.style.BaseStyles;
 
 /**
  * A view allowing admins to import a JSON representation of a program and add that program to their
  * environment.
  */
 public class AdminImportView extends BaseHtmlView {
   /**
-   * Play Framework can only parse request bodies up to 100KB: See
-   * https://www.playframework.com/documentation/2.4.x/ScalaBodyParsers#Max-content-length and
-   * https://github.com/civiform/civiform/issues/816. So, set the max length to be slightly under.
+   * Play Framework defaults to 100KB memory limit per request. We set this to 256KB in
+   * application.conf to allow large programs to be imported. More info:
+   * https://www.playframework.com/documentation/2.9.x/JavaBodyParsers#Content-length-limits
    *
-   * <p>Note that the HTML textarea element will automatically truncate the string to be the max
-   * character length, which will likely result in JSON parsing errors. TODO(#7087): Make that
-   * truncation obvious to admins.
+   * <p>Pasting in a json string above this limit will automatically truncate the string which
+   * results in a server error.
    */
-  private static final int MAX_TEXT_LENGTH = 100000;
+  private static final int MAX_TEXT_LENGTH = 256000;
```

Comment:
I tried adding a test for this case (what a pain) but the json string was so large that the formatter couldn't handle it. I thought about it a little more and I'm not sure we need to test the case where we're submitting a super large program since we would just be testing that these limit settings work, which is not logic we control. But I'm curious what you both think about adding a test case and, if you think we need it, how I might be able to test it?